Lab_2.ipynb

```diff
@@ -0,0 +1,783 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# average cosmic ray background\n",
+    "crb = 2\n",
+    "# average number of gamma rays\n",
+    "Y = 9\n",
+    "\n",
+    "x_1=[]\n",
+    "avg_crb=[]\n",
+    "for i in range(0, 15):\n",
+    "    x_1.append(i)\n",
+    "    avg_crb.append(stats.poisson.pmf(i, crb))\n",
+    "    \n",
+    "plt.step(x_1, avg_crb)\n",
+    "plt.title('Cosmic Ray Background Probability Density')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 87,
+   "id": "6963e637-c3df-473a-b4cf-74ba44c29f68",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def mult_conv(pdf, n):\n",
```

Comment:
Good job in putting this into a single well-named function, it really helps improve readability and makes it easy to follow your code!